src/Schema/aspire-8.0.json

```diff
@@ -73,6 +73,47 @@
               "bindMounts": {
                 "$ref": "#/definitions/bindMounts"
               },
+              "volumes": {
+                "$ref": "#/definitions/volumes"
+              },
+              "build": false
+            },
+            "additionalProperties": false
+          },
+          {
+            "type": "object",
+            "description": "A generic container resource.",
+            "required": [ "type", "image", "build" ],
+            "properties": {
+              "type": {
+                "const": "container.v1"
+              },
+              "image": {
+                "type": "string",
+                "description": "A string representing the container image to be used."
+              },
```

Comment:
is it still valid to have `image` for container.v1?  It is marked as required, so, I'm a little confused about it.

Reply:
@mitchdenny , can you confirm if `image` is actually used to give a name for the image that is created from the `build` field?  For example

```json
{
   "image": "regystry/my-image:v1",
   "build": {
        "path": "some/docker/context/path"
   }
}
```

azd would build the dockerfile in the `path` mentioned on the `build` field and use the name/tag `regystry/my-image:v1` for it?   Which means that, the same image is overridden on each deploy until the `image` name is updated in the AppHost to, say, `regystry/my-image:v2` ??

Reply:
@vhvb1989 the `image` field is still required and determines the name of the built image. When it comes to deployment using AZD I think it might make sense to push the image to the registry with the repo name being the image name but for the container app to reference the image via SHA?

Reply:
@vhvb1989 following the meeting that we had today I've changed the schema so that `image` and `build` are exclusive fields on `container.v1`. The plan is to reuse the container image naming logic that you use for `project.v0` resources.